lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -233,34 +233,33 @@
 
     // CASE: empty dashboard
     .empty-dashboard-message {
+      border: 3px solid $gray-l4;
+      background: $gray-l6;
       padding: ($baseline*2) 0;
       text-align: center;
 
       p {
         color: $lighter-base-font-color;
-        font-style: italic;
+        font-size: x-large;
```

Comment:
NIT: could we use a _typography.scss extend here instead to achieve the same result?